pkg/reconciler/instances/istio/action.go

```diff
@@ -1,37 +1,104 @@
 package istio
 
 import (
-	"fmt"
+	"bytes"
 	"github.com/kyma-incubator/reconciler/pkg/reconciler"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/chart"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/file"
 	"github.com/kyma-incubator/reconciler/pkg/reconciler/service"
 	"github.com/pkg/errors"
+	"io"
+	"os"
 	"os/exec"
+	"strings"
 )
 
 const (
-	istioctl1_10_2 = "/bin/istioctl-1.10.2"
+	istioctl1_11_1      = "/bin/istioctl-1.11.1" // change path to the local `istioctl` if debugging locally
+	yaml_delimiter      = "---"
+	istio_operator_kind = "kind: IstioOperator"
 )
 
 type ReconcileAction struct {
 }
 
 func (a *ReconcileAction) Run(version, profile string, config []reconciler.Configuration, context *service.ActionContext) error {
-	ws, err := context.WorkspaceFactory.Get(version)
+	var overrides = make(map[string]interface{}, len(config))
+
+	for _, configEntry := range config {
+		overrides[configEntry.Key] = configEntry.Value
+	}
+
+	comp := chart.NewComponent("istio-configuration", "istio-system", overrides)
+	componentSet := chart.NewComponentSet(context.KubeClient.Kubeconfig(), version, profile, []*chart.Component{comp})
```

Comment:
Depending if #121 gets merged before, this part will need some minor change (if you have luck and you PR goes in first, no changes will be needed :) ).